src/renderer/entities/proxy/model/proxy-model.ts

```diff
@@ -191,4 +191,7 @@ export const proxyModel = {
     proxyGroupsUpdated,
     proxyGroupsRemoved,
   },
+  output: {
+    proxyStatusUpdated: merge([addProxiesFx, removeProxiesFx]),
```

Comment:
I see handling for adding wallets/accounts + remove accounts but nothing for removing the wallet
This part of code is trying to cover that case of wallet deletion?
Explicit worker call after wallet deletion causes problems?

Reply:
This one should cover a case when we add or remove a proxy from an existing wallet. Otherwise, if we don't run a proxy worker after that - the list of proxies won't change

Reply:
Actually when we add proxy through details we manually add it into the store/db
This was done to avoid running worker
Maybe it's bugged, I'll take a look